addons/website/data/website_data.xml

```diff
@@ -166,17 +166,14 @@
                         <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
                             <div class="container s_allow_columns">
                                 <div class="row">
-                                    <div class="col-lg-7 col-xl-6 me-lg-auto">
-                                        <span class="d-block fa fa-4x fa-thumbs-up mx-auto rounded-circle bg-primary"/><br/>
-                                        <h1 class="text-center">Thank You!</h1>
-                                        <div class="s_hr pb16 pt16" data-snippet="s_hr" data-name="Separator">
-                                            <hr class="w-50 mx-auto border-dark"/>
+                                    <div class="col-lg-7 col-xl-6 me-lg-auto text-center">
+                                        <div class="d-inline-block mx-auto p-4">
+                                            <i class="fa fa-paper-plane fa-2x mb-3 rounded-circle text-bg-success" role="img" aria-label="Success" title="Success"/>
+                                            <h1 class="fw-bolder">Thank You!</h1>
+                                            <p class="lead mb-0">Your message has been sent.</p>
+                                            <p class="lead">We will get back to you shortly.</p>
+                                            <a href="/" title="Go to Homepage">Go to Homepage</a>
```

Comment:
Definitely remove this `title` attribute, it is the same as the content of the link... :grimacing: